retrofit-converter-jspoon/build.gradle

```diff
@@ -8,6 +8,11 @@ plugins {
 dependencies {
     compile 'com.squareup.retrofit2:retrofit:2.3.0'
     compile "pl.droidsonroids:jspoon:${POM_VERSION}"
+
+    testCompile 'junit:junit:4.12'
```

Comment:
We should extract all the version numbers to top-level `build.gradle` file and replace hardcoded numbers by those variables. Not only in this file. But it can be done in a separate PR i think.